che-theia-plugins.yaml

```diff
@@ -547,7 +547,9 @@ plugins:
           path: /home/theia/.m2
     extensions:
       - >-
-        https://github.com/SonarSource/sonarlint-vscode/releases/download/1.16.0/sonarlint-vscode-1.16.0.vsix
+        https://github.com/SonarSource/sonarlint-vscode/releases/download/1.20.1/sonarlint-vscode-1.20.1.vsix
+      - >-
+        https://open-vsx.org/api/vscode/typescript-language-features/1.54.1/file/vscode.typescript-language-features-1.54.1.vsix
```

Comment:
For example: https://github.com/eclipse/che-plugin-registry/blob/master/che-theia-plugins.yaml#L45

Reply:
No need to use the latest version, set the current theia version of the `typescript-language-features` which works with the new `sonarlint` plugin.

Reply:
I still don't get why typescript is a requirement for sonarlint
It looks like more it should be an optional dependency

If I have a java project I don't see why typescript would be there ?

Reply:
I thought it was the outcome of https://github.com/eclipse/che/issues/16589

Reply:
it's optional dependency
Like to use sonarlint on a JavaScript project you need to add typescript and for java you need to add vscode-java but it should come by default

Reply:
Okay @vinokurig let's remove it then

Reply:
It's not an optional dependency: https://github.com/SonarSource/sonarlint-vscode/blob/683ce4772b0c94b6e58e8dc39832d39b2465a64c/package.json#L45-L47
And the plugin doesn't work without it:
![screenshot-codeready-codeready-workspaces-operator apps sandbox x8i5 p1 openshiftapps com-2021 03 12-08_47_40](https://user-images.githubusercontent.com/7668752/110903730-8e986700-8310-11eb-8a24-21677574675b.png)

Reply:
ok it's a new dependency :-/ 
I assume they did that as in VS Code the typescript-language-features is always there.

Would have be cool if we were supporting 'dependencies' as well by just telling sonarlint was depending on typescript and then it would have install typescript in addition to sonarlint.

Reply:
Agreed, is there an issue open for that? If not, please file one